src/shield/rules.ts

```diff
@@ -21,3 +21,23 @@ export const isBookingCreator = rule({ cache: "contextual" })(
     return currentUser.id === bookingUser.id;
   }
 );
+
+export const isEventCreator = rule({ cache: "contextual" })(
+  async (parent, args, ctx: AppContext) => {
+    const currentUser: User = ctx.auth.user;
+    const eventOrganiser: User = await ctx.prisma
+      .event({ id: args.id })
+      .organiser();
+    return currentUser.id === eventOrganiser.id;
+  }
+);
+
+export const isCommentCreator = rule({ cache: "contextual" })(
+  async (parent, args, ctx: AppContext) => {
+    const currentUser: User = ctx.auth.user;
+    const commentCreator: User = await ctx.prisma
+      .comment({ id: args.commentId })
```

Comment:
I noticed that the `args` object returns `commentId` instead of just a simple `id`. Is there an advantage to using `commentId` as opposed to `id` on the graphql api for a client app?

Reply:
there is no difference actually, did that purely for the purpose of clarity

Reply:
I think I will agree with @98sean98 on using just a simple `id`. It is being used in the `deleteComment` mutation, and for the sake of consistency, why do we just not use `id` just like other mutations.

It seems quite clear that `deleteComment` is going to deal with `id` of comment. Just my opinion. But I think @welvin21 has been outnumbered in terms of majority 🤣 .

Reply:
😂 let me change that in a moment